gems/core/tasks.py

```diff
@@ -127,4 +129,31 @@ def export_data():
     except Exception as ex:
         logger.error('export_data[Task] failed: %s' % ex.message)
 
-    logger.info('export_data[Task] :: Completed')
\ No newline at end of file
+    logger.info('export_data[Task] :: Completed')
+
+
+@task(bind=True)
+def import_contacts():
+    logger.info('importing contacts :: Started')
+
+    api = None
+    try:
+        api = ContactsApiClient(settings.VUMI_TOKEN)
+    except Exception:
```

Comment:
This `except Exception:` absolutely anything that can go wrong and assumes the problem is that it cannot connect to Vumi. Please specify the exact `Exception` you're wanting to catch here. It may just be easier to let this task crash and burn when it does fail, the error should the go to the configured Sentry instance.